CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Fleet 4.48.3 (Apr 16, 2024)
+
+### Bug fixes
+
+* Calendar webhook will retry if it receives response 429 Too Many Requests. Webhook request will retry for 30 minutes with a 1 minute max delay between retries.
+* Updated label endpoints and UI to prevent creating, updating, or deleting built-in labels.
+* Fleet UI: Fixed edge cases of team ID being lost in various flows
```

Comment:
Remove "Fleet UI:" prefix. ChatGPT does this sometimes for no reason. Otherwise it's good.